docs/concepts/07_dip/05_dapp_developer.md

```diff
@@ -26,8 +26,128 @@ Import the SDK into your code:
+
+The method returns the DID base proof. You have to call a second method, the `[generateDipSubmittableExtrinsic](https://kiltprotocol.github.io/dip-sdk/functions/generateDipSubmittableExtrinsic.html)` method to generate a submittable extrinsic.
+
+You need to pass the following parameters:
+
+-   The API of the consumer chain.
+-   The base proof.
+-   The extrinsic call to the consumer chain.
+-   The DID URI.
+
+:::info Submittable extrinsics
+
+A transaction that originates from an external account and affects the state of the blockchain.
+An extrinisc executes actions on the network, such as transferring funds, making governance decisions, using functionality of the parachain, or interacting with smart contracts.
+
+:::
+
+```typescript
+const dipSubmittable = DipSdk.generateDipSubmittableExtrinsic({
+    api: consumerApi,
+    baseDipProof,
+    call,
+    didUri: did.uri,
+})
+```
+
+The method returns the different components of the proof, [which you can see in the example code](https://github.com/KILTprotocol/dip-sdk/blob/9ad141b3757e076744ab8b2d29bcf10bbeaddd9f/tests/dip-provider-template-dip-consumer-template/develop.test.ts#L219):
+
+-   The provider head proof, which is proof of the provider parachain header on the relay chain.
+-   The commitment proof, which proves the deep commitment for a specific subject, which is the DID URI.
+-   The actual deep proof, which reveals parts of the DID document as specified by key IDs, proof version, whether to include the web rename and the linked account, and the proof version that the user specified.
+
+Behind the scenes, the method uses the `dispatchAs` method ([and corresponding chain method](https://github.com/KILTprotocol/kilt-node/blob/4ddb8a0ef6258873458f19d3ee9dcb6d7c24e645/pallets/did/src/lib.rs#L1152)) to pass the extrinsic following the consumer's type registry.
+You can now sign and submit to the consumer chain.
+
+```typescript
+await signAndSubmitTx(consumerApi, dipSubmittable, submitterKeypair)
+```
+
+### 3. Linking accounts (optional)
+
+Linked accounts let you specify which accounts you want to prove that you control when you make the cross-chain proof. As part of the proof provided, you can also include other values, such as the web3name.
+
+For all the accounts you want to link, use the `associateAccountToChainArgs` method, [as detailed in this guide](../../develop/01_sdk/02_cookbook/03_account_linking/01_link.md##linking-an-account-to-a-did).
+
+You can then batch all the linked account transactions and authorize them using the `authorizeTx` method.
+
+```typescript
+const signedLinkedAccounts = await Kilt.Did.authorizeTx(
+    newFullDidUri,
+    providerApi.tx.utility.batchAll(linkAccountTxs),
+    signCallback,
+    newSubmitterKeypair.address as KiltAddress,
+    { txCounter: new BN(4) }
+)
+```
+
+:::tip
+
+You can also include accounts you want to link as part of the initial configuration object for the base proof as a list of account addresses.
```

Comment:
This is wrong. In the config object, you only specify which of the accounts that are **already** linked you want to reveal in the proof. It does not link the accounts. In the tests I linked the accounts so that they could be revealed in the proof.